### PR TITLE
feat(i18n-SI): adds Slovenian translations

### PR DESCRIPTION
**Pull request by jernejp21**
Added Slovenian translation for Fluidd.

Signed-off-by: Jernej Pangerc <jernej.pangerc@tutamail.com>

### CODE REVIEW

File: src/globals.ts [hunk 1]
```diff
@@ -434,6 +434,7 @@ export const SupportedLocales = Object.freeze([
   { name: 'Nederlands', code: 'nl' },
   { name: 'Português', code: 'pt' },
   { name: 'Русский', code: 'ru' },
+  { name: 'slovenščina', code: 'si' },
```
Comment (pedrolamas) [inline src/globals.ts:437]: [According to ISO 639-1](https://www.loc.gov/standards/iso639-2/php/langcodes_name.php?iso_639_1=sl), the Slovenian identifier should be "sl" and not "si"

```suggestion
  { name: 'slovenščina', code: 'sl' },
```

Comment (jernejp21) [reply src/globals.ts:437]: Thank you. I will change this and also rename locale file from si.yaml to sl.yaml

File: src/globals.ts [hunk 2]
```diff
@@ -434,6 +434,7 @@ export const SupportedLocales = Object.freeze([
   { name: 'Nederlands', code: 'nl' },
   { name: 'Português', code: 'pt' },
   { name: 'Русский', code: 'ru' },
+  { name: 'slovenščina', code: 'sl' },
```
Comment (pedrolamas) [inline src/globals.ts:437]: Minor change here, just getting this capitalized.

```suggestion
  { name: 'Slovenščina', code: 'sl' },
```

**Review by pedrolamas — APPROVED** [review]
Thank you @jernejp21

**Review by matmen — APPROVED** [review]

**Pull request merged**